tez-dag/src/main/java/org/apache/tez/dag/app/dag/impl/VertexImpl.java

```diff
@@ -4930,4 +4981,19 @@ public boolean getTaskRescheduleRelaxedLocality() {
   public Map<String, Set<String>> getDownstreamBlamingHosts(){
     return downstreamBlamingHosts;
   }
+
+  /**
+   * Initialize context from vertex shuffle deletion.
+   * @param deletionHeight
+   */
+  public void initShuffleDeletionContext(int deletionHeight) {
+    VertexShuffleDataDeletionContext vShuffleDeletionContext = new VertexShuffleDataDeletionContext(deletionHeight);
+    vShuffleDeletionContext.setSpannedVertices(this);
+    this.vShuffleDeletionContext = vShuffleDeletionContext;
+  }
+
+  @VisibleForTesting
+  public VertexShuffleDataDeletionContext getVShuffleDeletionContext() {
```

Comment:
this is not used currently in any of the tests
I'm fine with leaving this part uncovered, can you please remove this method?

Reply:
This is used by testVertexShuffleDelete() in TestVertexImpl